{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Product defects","children":[{"name":"Timed out after 60.000s.\nExpected\n    <bool>: false\nto be true","children":[{"name":"[It] Project CR test Project Deletion Should Delete an existing project successfully","uid":"bef8a23857ce5557","parentUid":"cad0674b2306dd77b62a2ba0a795baef","status":"failed","time":{"start":1663214989000,"stop":1663215110347,"duration":121347},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"[It] Project CR test Project Creation Should update Project while applying valid manifest with existing Project name","uid":"dbe0183296ffa8e9","parentUid":"cad0674b2306dd77b62a2ba0a795baef","status":"failed","time":{"start":1663214989000,"stop":1663215063833,"duration":74833},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"[It] Project CR test Project Creation Should create Project while using valid manifest","uid":"8c916ecd6ba60358","parentUid":"cad0674b2306dd77b62a2ba0a795baef","status":"failed","time":{"start":1663214989000,"stop":1663215067658,"duration":78658},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"[It] Project CR test Project Update Should update successfully when Project Applied while removing a service account name in Write users","uid":"92d2f26bdcd9534b","parentUid":"cad0674b2306dd77b62a2ba0a795baef","status":"failed","time":{"start":1663214989000,"stop":1663215056723,"duration":67723},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"[It] Project CR test Project Update Should update successfully when Project Applied while removing a service account name in Read users","uid":"d51eba5901130a13","parentUid":"cad0674b2306dd77b62a2ba0a795baef","status":"failed","time":{"start":1663214989000,"stop":1663215055716,"duration":66716},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"[It] Project CR test Project Deletion Should fail when deleting a project that does not exist","uid":"b91faa6370231130","parentUid":"cad0674b2306dd77b62a2ba0a795baef","status":"failed","time":{"start":1663214989000,"stop":1663215056574,"duration":67574},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"[It] Project CR test Project Update Should fail when Project is Applied with service account name as combination of special characters in Read users","uid":"ef05d1b9c2bccd78","parentUid":"cad0674b2306dd77b62a2ba0a795baef","status":"failed","time":{"start":1663214989000,"stop":1663215110413,"duration":121413},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"cad0674b2306dd77b62a2ba0a795baef"},{"name":"Timed out after 60.001s.\nExpected\n    <bool>: false\nto be true","children":[{"name":"[It] Project CR test Project Update Should update successfully when Project applied with valid service account name in Write users","uid":"49c1ffb1f3e04a80","parentUid":"bdb6f7322995205e885b5ea7481ccb7f","status":"failed","time":{"start":1663214989000,"stop":1663215055906,"duration":66906},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"[It] Project CR test Project Update Should fail when Project is Applied with service account name as combination of special characters in Write users","uid":"6ed9201fabe7ce7d","parentUid":"bdb6f7322995205e885b5ea7481ccb7f","status":"failed","time":{"start":1663214989000,"stop":1663215110373,"duration":121373},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"[It] Project CR test Project Update Should Update successfullyy when Project is Applied with valid service account name in Read users","uid":"5177f72f1583e227","parentUid":"bdb6f7322995205e885b5ea7481ccb7f","status":"failed","time":{"start":1663214989000,"stop":1663215110545,"duration":121545},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"[It] Project CR test Project Update Should fail when Project is Applied with service account name as blank in Read users","uid":"127f340e534c675a","parentUid":"bdb6f7322995205e885b5ea7481ccb7f","status":"failed","time":{"start":1663214989000,"stop":1663215055750,"duration":66750},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"[It] Project CR test Project Creation Should create Multiple Projects in controller using valid manifest","uid":"9004ea9c56ff3468","parentUid":"bdb6f7322995205e885b5ea7481ccb7f","status":"failed","time":{"start":1663214989000,"stop":1663215050433,"duration":61433},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"bdb6f7322995205e885b5ea7481ccb7f"},{"name":"Unexpected error:\n    <*shell.ErrWithCmdOutput | 0xc0006a5578>: {\n        Underlying: <*exec.ExitError | 0xc000591d20>{\n            ProcessState: {\n                pid: 11827,\n                status: 256,\n                rusage: {\n                    Utime: {Sec: 0, Usec: 236808},\n                    Stime: {Sec: 0, Usec: 72246},\n                    Maxrss: 80092,\n                    Ixrss: 0,\n                    Idrss: 0,\n                    Isrss: 0,\n                    Minflt: 13192,\n                    Majflt: 0,\n                    Nswap: 0,\n                    Inblock: 0,\n                    Oublock: 12880,\n                    Msgsnd: 0,\n                    Msgrcv: 0,\n                    Nsignals: 0,\n                    Nvcsw: 296,\n                    Nivcsw: 289,\n                },\n            },\n            Stderr: nil,\n        },\n        Output: {\n            stdout: {\n                Lines: nil,\n                merged: {\n                    Mutex: {state: 0, sema: 0},\n                    Lines: [\n                        \"Error from server (InternalError): error when creating \\\"/tmp/Cluster%20CR%20tests%20Cluster%20CR%20validation%20Creates%20cluster%20secrets2879192980\\\": Internal error occurred: failed calling webhook \\\"mproject.kb.io\\\": failed to call webhook: Post \\\"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-project?timeout=10s\\\": dial tcp 10.96.205.182:443: connect: connection refused\",\n                    ],\n                },\n            },\n            stderr: {\n                Lines: [\n                    \"Error from server (InternalError): error when creating \\\"/tmp/Cluster%20CR%20tests%20Cluster%20CR%20validation%20Creates%20cluster%20secrets2879192980\\\": Internal error occurred: failed calling webhook \\\"mproject.kb.io\\\": failed to call webhook: Post \\\"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-project?timeout=10s\\\": dial tcp 10.96.205.182:443: connect: connection refused\",\n                ],\n                merged: {\n                    Mutex: {state: 0, sema: 0},\n                    Lines: [\n                        \"Error from server (InternalError): error when creating \\\"/tmp/Cluster%20CR%20tests%20Cluster%20CR%20validation%20Creates%20cluster%20secrets2879192980\\\": Internal error occurred: failed calling webhook \\\"mproject.kb.io\\\": failed to call webhook: Post \\\"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-project?timeout=10s\\\": dial tcp 10.96.205.182:443: connect: connection refused\",\n                    ],\n                },\n            },\n            merged: {\n                Mutex: {state: 0, sema: 0},\n                Lines: [\n                    \"Error from server (InternalError): error when creating \\\"/tmp/Cluster%20CR%20tests%20Cluster%20CR%20validation%20Creates%20cluster%20secrets2879192980\\\": Internal error occurred: failed calling webhook \\\"mproject.kb.io\\\": failed to call webhook: Post \\\"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-project?timeout=10s\\\": dial tcp 10.96.205.182:443: connect: connection refused\",\n                ],\n            },\n        },\n    }\n    error while running command: exit status 1; Error from server (InternalError): error when creating \"/tmp/Cluster%20CR%20tests%20Cluster%20CR%20validation%20Creates%20cluster%20secrets2879192980\": Internal error occurred: failed calling webhook \"mproject.kb.io\": failed to call webhook: Post \"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-project?timeout=10s\": dial tcp 10.96.205.182:443: connect: connection refused\noccurred","children":[{"name":"[It] Cluster CR tests Cluster CR validation Creates cluster secrets","uid":"5de0605e3533bf18","parentUid":"213855d3118d12138bd46c3f0189ae1f","status":"failed","time":{"start":1663214989000,"stop":1663214989386,"duration":386},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"213855d3118d12138bd46c3f0189ae1f"},{"name":"Unexpected error:\n    <*shell.ErrWithCmdOutput | 0xc0003b3638>: {\n        Underlying: <*exec.ExitError | 0xc0003930a0>{\n            ProcessState: {\n                pid: 10873,\n                status: 256,\n                rusage: {\n                    Utime: {Sec: 0, Usec: 211116},\n                    Stime: {Sec: 0, Usec: 113377},\n                    Maxrss: 83952,\n                    Ixrss: 0,\n                    Idrss: 0,\n                    Isrss: 0,\n                    Minflt: 15546,\n                    Majflt: 0,\n                    Nswap: 0,\n                    Inblock: 0,\n                    Oublock: 12880,\n                    Msgsnd: 0,\n                    Msgrcv: 0,\n                    Nsignals: 0,\n                    Nvcsw: 494,\n                    Nivcsw: 501,\n                },\n            },\n            Stderr: nil,\n        },\n        Output: {\n            stdout: {\n                Lines: nil,\n                merged: {\n                    Mutex: {state: 0, sema: 0},\n                    Lines: [\n                        \"Error from server (InternalError): error when applying patch:\",\n                        \"{\\\"status\\\":{}}\",\n                        \"to:\",\n                        \"Resource: \\\"controller.kubeslice.io/v1alpha1, Resource=projects\\\", GroupVersionKind: \\\"controller.kubeslice.io/v1alpha1, Kind=Project\\\"\",\n                        \"Name: \\\"projectupdatetest\\\", Namespace: \\\"kubeslice-controller\\\"\",\n                        \"for: \\\"/tmp/Project%20CR%20test%20Project%20Update%20Should%20fail%20when%20Project%20is%20applied%20with%20service%20account%20name%20as%20blank%20in%20Write%20users3571371373\\\": error when patching \\\"/tmp/Project%20CR%20test%20Project%20Update%20Should%20fail%20when%20Project%20is%20applied%20with%20service%20account%20name%20as%20blank%20in%20Write%20users3571371373\\\": Internal error occurred: failed calling webhook \\\"mproject.kb.io\\\": failed to call webhook: Post \\\"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-project?timeout=10s\\\": dial tcp 10.96.205.182:443: connect: connection refused\",\n                    ],\n                },\n            },\n            stderr: {\n                Lines: [\n                    \"Error from server (InternalError): error when applying patch:\",\n                    \"{\\\"status\\\":{}}\",\n                    \"to:\",\n                    \"Resource: \\\"controller.kubeslice.io/v1alpha1, Resource=projects\\\", GroupVersionKind: \\\"controller.kubeslice.io/v1alpha1, Kind=Project\\\"\",\n                    \"Name: \\\"projectupdatetest\\\", Namespace: \\\"kubeslice-controller\\\"\",\n                    \"for: \\\"/tmp/Project%20CR%20test%20Project%20Update%20Should%20fail%20when%20Project%20is%20applied%20with%20service%20account%20name%20as%20blank%20in%20Write%20users3571371373\\\": error when patching \\\"/tmp/Project%20CR%20test%20Project%20Update%20Should%20fail%20when%20Project%20is%20applied%20with%20service%20account%20name%20as%20blank%20in%20Write%20users3571371373\\\": Internal error occurred: failed calling webhook \\\"mproject.kb.io\\\": failed to call webhook: Post \\\"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-project?timeout=10s\\\": dial tcp 10.96.205.182:443: connect: connection refused\",\n                ],\n                merged: {\n                    Mutex: {state: 0, sema: 0},\n                    Lines: [\n                        \"Error from server (InternalError): error when applying patch:\",\n                        \"{\\\"status\\\":{}}\",\n                        \"to:\",\n                        \"Resource: \\\"controller.kubeslice.io/v1alpha1, Resource=projects\\\", GroupVersionKind: \\\"controller.kubeslice.io/v1alpha1, Kind=Project\\\"\",\n                        \"Name: \\\"projectupdatetest\\\", Namespace: \\\"kubeslice-controller\\\"\",\n                        \"for: \\\"/tmp/Project%20CR%20test%20Project%20Update%20Should%20fail%20when%20Project%20is%20applied%20with%20service%20account%20name%20as%20blank%20in%20Write%20users357137...\n\nGomega truncated this representation as it exceeds 'format.MaxLength'.\nConsider having the object provide a custom 'GomegaStringer' representation\nor adjust the parameters in Gomega's 'format' package.\n\nLearn more here: https://onsi.github.io/gomega/#adjusting-output\n\n    error while running command: exit status 1; Error from server (InternalError): error when applying patch:\n    {\"status\":{}}\n    to:\n    Resource: \"controller.kubeslice.io/v1alpha1, Resource=projects\", GroupVersionKind: \"controller.kubeslice.io/v1alpha1, Kind=Project\"\n    Name: \"projectupdatetest\", Namespace: \"kubeslice-controller\"\n    for: \"/tmp/Project%20CR%20test%20Project%20Update%20Should%20fail%20when%20Project%20is%20applied%20with%20service%20account%20name%20as%20blank%20in%20Write%20users3571371373\": error when patching \"/tmp/Project%20CR%20test%20Project%20Update%20Should%20fail%20when%20Project%20is%20applied%20with%20service%20account%20name%20as%20blank%20in%20Write%20users3571371373\": Internal error occurred: failed calling webhook \"mproject.kb.io\": failed to call webhook: Post \"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-project?timeout=10s\": dial tcp 10.96.205.182:443: connect: connection refused\noccurred","children":[{"name":"[It] Project CR test Project Update Should fail when Project is applied with service account name as blank in Write users","uid":"6b0e551a14628d0","parentUid":"2d7c2e94dcd93e69cd89d12032cb7860","status":"failed","time":{"start":1663214989000,"stop":1663215049533,"duration":60533},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"2d7c2e94dcd93e69cd89d12032cb7860"},{"name":"Unexpected error:\n    <*shell.ErrWithCmdOutput | 0xc000430288>: {\n        Underlying: <*exec.ExitError | 0xc0006e20a0>{\n            ProcessState: {\n                pid: 11836,\n                status: 256,\n                rusage: {\n                    Utime: {Sec: 0, Usec: 222415},\n                    Stime: {Sec: 0, Usec: 91349},\n                    Maxrss: 81684,\n                    Ixrss: 0,\n                    Idrss: 0,\n                    Isrss: 0,\n                    Minflt: 13439,\n                    Majflt: 0,\n                    Nswap: 0,\n                    Inblock: 0,\n                    Oublock: 12880,\n                    Msgsnd: 0,\n                    Msgrcv: 0,\n                    Nsignals: 0,\n                    Nvcsw: 312,\n                    Nivcsw: 312,\n                },\n            },\n            Stderr: nil,\n        },\n        Output: {\n            stdout: {\n                Lines: nil,\n                merged: {\n                    Mutex: {state: 0, sema: 0},\n                    Lines: [\n                        \"Error from server (InternalError): error when creating \\\"/tmp/Cluster%20negative%20tests%20Worker%20Cluster%20Registration%20with%20Wrong%20endpoint2667703893\\\": Internal error occurred: failed calling webhook \\\"mproject.kb.io\\\": failed to call webhook: Post \\\"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-project?timeout=10s\\\": dial tcp 10.96.205.182:443: connect: connection refused\",\n                    ],\n                },\n            },\n            stderr: {\n                Lines: [\n                    \"Error from server (InternalError): error when creating \\\"/tmp/Cluster%20negative%20tests%20Worker%20Cluster%20Registration%20with%20Wrong%20endpoint2667703893\\\": Internal error occurred: failed calling webhook \\\"mproject.kb.io\\\": failed to call webhook: Post \\\"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-project?timeout=10s\\\": dial tcp 10.96.205.182:443: connect: connection refused\",\n                ],\n                merged: {\n                    Mutex: {state: 0, sema: 0},\n                    Lines: [\n                        \"Error from server (InternalError): error when creating \\\"/tmp/Cluster%20negative%20tests%20Worker%20Cluster%20Registration%20with%20Wrong%20endpoint2667703893\\\": Internal error occurred: failed calling webhook \\\"mproject.kb.io\\\": failed to call webhook: Post \\\"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-project?timeout=10s\\\": dial tcp 10.96.205.182:443: connect: connection refused\",\n                    ],\n                },\n            },\n            merged: {\n                Mutex: {state: 0, sema: 0},\n                Lines: [\n                    \"Error from server (InternalError): error when creating \\\"/tmp/Cluster%20negative%20tests%20Worker%20Cluster%20Registration%20with%20Wrong%20endpoint2667703893\\\": Internal error occurred: failed calling webhook \\\"mproject.kb.io\\\": failed to call webhook: Post \\\"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-project?timeout=10s\\\": dial tcp 10.96.205.182:443: connect: connection refused\",\n                ],\n            },\n        },\n    }\n    error while running command: exit status 1; Error from server (InternalError): error when creating \"/tmp/Cluster%20negative%20tests%20Worker%20Cluster%20Registration%20with%20Wrong%20endpoint2667703893\": Internal error occurred: failed calling webhook \"mproject.kb.io\": failed to call webhook: Post \"https://kubeslice-controller-webhook-service.kubeslice-controller.svc:443/mutate-controller-kubeslice-io-v1alpha1-project?timeout=10s\": dial tcp 10.96.205.182:443: connect: connection refused\noccurred","children":[{"name":"[It] Cluster negative tests Worker Cluster Registration with Wrong endpoint","uid":"a3031fb14c7f8c6d","parentUid":"c913928ad0723c20aab3e4ec1247360a","status":"failed","time":{"start":1663214989000,"stop":1663214989578,"duration":578},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"c913928ad0723c20aab3e4ec1247360a"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}],"name":"categories"}